webapp/app/components/MessageBox/MessageBox.component.js

```diff
@@ -0,0 +1,48 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import { Wrapper, Field } from './MessageBox.styles';
+import { UserAvatar } from '../UserAvatar/UserAvatar.component';
+
+
+export class MessageBox extends PureComponent {
+  static propTypes = {
+    message: PropTypes.string,
+    onCreateMessage: PropTypes.func.isRequired,
+  };
+
+  state = {
+    message: '',
+  };
+
+  onChange = (event) => {
```

Comment:
onChange = ({ target: { value: message } }) => this.setState({ message  });

Wiesz ze nie przepadam za 'wąsami'

Reply:
ale ogolnie to tez mi obojetnie :)